Gruntfile.js

```diff
@@ -362,12 +362,18 @@ module.exports = function(grunt) {
   grunt.registerTask("dist-compile", ["test", "uglify", "compress"]);
 
   grunt.registerTask("commitjs", ["dist-compile", "gitcommit:built"]);
-  grunt.registerTask("default", ["connect", "dev-compile", "watch"]);
+  grunt.registerTask("default", ["connect", "dev-compile", "watch-silent"]);
 
   grunt.registerTask("test", ["dev-compile", "test-local"]);
   grunt.registerTask("test-local", ["blanket_mocha", "ts:verifyDefinitionFiles", "lint"]);
   grunt.registerTask("test-sauce", ["connect", "saucelabs-mocha"]);
 
+  grunt.registerTask("watch-silent", function() {
+    // HACKHACK #2508
+    grunt.log.header = function() {};
+    grunt.task.run(["watch"]);
```

Comment:
Yeah, it's in the description, with link to what I want to do and everything

This is simply removing the Running "foo:bar" task, for watch only. The header is still used for all our CI stuff.
Basically, wrapper around the watch which also disables the headers